src/components/cylc/Toolbar.vue

```diff
@@ -0,0 +1,150 @@
+          dark
+          icon
+          @click.stop="onClickBtn"
+        >
+          <v-icon>mdi-view-list</v-icon>
+        </v-btn>
+        <!-- title -->
+        <span class="c-toolbar-title">{{ title }}</span>
+      </v-toolbar-title>
+    </div>
+
+    <!-- control bar elements displayed only when a suite has been positioned -->
+    <template v-if="tree && tree.length > 0">
+      <a @click="onClickPause">
+        <v-icon color="#5E5E5E">mdi-pause</v-icon>
+      </a>
+
+      <a @click="onClickStop">
+        <v-icon color="#5E5E5E">mdi-stop</v-icon>
+      </a>
+
+      <a>
+        <v-chip color="#E7E7E7" @click="toggleExtended">Control</v-chip>
+      </a>
+
+      <span>Running, will stop at 30000101T0000 cycle</span>
+
+      <v-spacer />
+
+      <a class="add-view" @click="onClickAddView">
+        Add View <v-icon color="#5995EB">mdi-plus-circle</v-icon>
+      </a>
+    </template>
+
+    <!-- displayed only when extended===true -->
+    <template v-slot:extension v-if="extended">
+      <span style="margin-left: 260px;">
+        <a @click="onClickPause">
+          <v-icon color="#5E5E5E">mdi-pause</v-icon>
+        </a>
+
+        <a @click="onClickStop">
+          <v-icon color="#5E5E5E">mdi-stop</v-icon>
+        </a>
+
+        <span>Other controls added in the future</span>
+      </span>
+    </template>
+
+  </v-toolbar>
+</template>
+
+<script>
+
+  import { mapMutations, mapState } from 'vuex'
+
+  export default {
+  data: () => ({
+    responsive: false,
+    responsiveInput: false,
+    extended: false
+  }),
+
+  computed: {
+    ...mapState('app', ['title']),
+    ...mapState('suites', ['tree'])
+  },
+
+  mounted () {
+    this.onResponsiveInverted()
+    window.addEventListener('resize', this.onResponsiveInverted)
+  },
+  beforeDestroy () {
+    window.removeEventListener('resize', this.onResponsiveInverted)
+  },
+
+  methods: {
+    ...mapMutations('app', ['setDrawer', 'toggleDrawer']),
+    onClickBtn () {
+      this.setDrawer(!this.$store.state.app.drawer)
+    },
+    onResponsiveInverted () {
+      if (window.innerWidth < 991) {
+        this.responsive = true
+        this.responsiveInput = false
+      } else {
+        this.responsive = false
+        this.responsiveInput = true
+      }
+    },
+    onClickPause () {
+      console.log("Pausing workflows has not been implemented yet")
+    },
+    onClickStop () {
+      console.log("Stopping workflows has not been implemented yet")
+    },
+    toggleExtended () {
+      this.extended = !this.extended
+    },
+    onClickAddView () {
+      console.log("Adding views has not been implemented yet")
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+  .v-toolbar {
+    min-height: inherit !important;
+    .v-toolbar__content {
+      min-height: inherit !important;
+    }
+  }
+
+  .c-toolbar {
+    background-color: #E7E7E7;
+    border-bottom: 3px solid #ccc;
+
+    .c-toolbar-title, .add-view {
+      font-size: 22px !important;
```

Comment:
mmm, why is `!important` needed here?

Reply:
I can investigate more tomorrow, but I believe it was due to conflicts with attributes from other classes of `a` set in vuetify theme 😓

Reply:
Dammit, is vuetify trying to dictate style of downstream components!?

Reply:
>Dammit, is vuetify trying to dictate style of downstream components!?

Not quite. I think the Toolbar is a component to be used, and not really extended. While it supports some level of extension via props, slots; I think styling is a bit limited.

>mmm, why is !important needed here?

Turns out this !important is not really necessary :tada: